src/fipp/edn.cljc

```diff
@@ -5,7 +5,24 @@
             [fipp.visit :refer [visit visit*]]
             [fipp.engine :refer (pprint-document)]))
 
-(defrecord EdnPrinter [print-meta symbols]
+(defn- ->lines [{:keys [print-level] :as printer} xform x]
+  (if (and print-level (neg? print-level))
+    "#"
+    (sequence xform x)))
+
+(defn- ->group [{:keys [print-length] :as printer} left-separator lines right-separator]
+  [:group left-separator
+          [:align lines]
+          (when (and print-length (<= print-length (count lines)))
+            " ...")
```

Comment:
Actually want `[:span :line "..."]` here, or more generally `[:span sep "..."]`. This change plus corresponding tests forthcoming.